packages/core/helper-plugin/src/utils/getFileExtension.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * Get the file extension of value passed in.
+ *
+ * This hook is used to create URL aware of the backend. Practical to resolve assets.
+ */
+export const getFileExtension = (ext: string) => (ext && ext[0] === '.' ? ext.substr(1) : ext);
```

Comment:
Please update test case as well

Reply:
I've migrated `getFileExtension.test.js` over to TypeScript.

But when I did that all of the Jest function were erroring as not defined.
I've searched for over an hour now online and tired updating jest configs but it only happens within the `helper-plugin` folder, so I've tried comparing other jest configs and jest preset files and even when adding code not included in helper plugins jest config, the eslint error still remains. So I've had to add `eslint-disable no-undef` to be able to commit the changes.
If you know of any way of fixing the eslint error please let me know and I'll make the update.

Reply:
> 

That worked thank you